src/render/solver.rs

```diff
@@ -507,8 +516,25 @@ async fn fetch_repo_data_records_with_progress(
     // Error out if an error occurred, but also update the progress bar
     let result = match result {
         Err(e) => {
+            let not_found = match &e {
+                FetchRepoDataError::HttpError(e) if e.status() == Some(StatusCode::NOT_FOUND) => {
+                    true
+                }
+                FetchRepoDataError::FailedToDownloadRepoData(e)
+                    if e.kind() == std::io::ErrorKind::NotFound =>
+                {
+                    true
+                }
+                _ => false,
+            };
```

Comment:
It would be nice if you could move this to `rattler` itself. Add a `FetchRepoDataError::NotFound`.

Reply:
Added that in https://github.com/mamba-org/rattler/pull/256 this pr now depends on that.